workflows/genome_annotation/lncRNAs_annotation/Galaxy-Workflow-lncRNAs_annotation_workflow-tests.yml

```diff
@@ -0,0 +1,40 @@
+- doc: Test outline for Galaxy-Workflow-lncRNAs_annotation_workflow.ga
+  job:
+    Genome assembly:
+      class: File
+      location: https://zenodo.org/records/11367439/files/genome_assembly.fasta
+      filetype: fasta
+    Genome annotation:
+      class: File
+      location: https://zenodo.org/records/11367439/files/genome_annotation.gff3
+      filetype: gff3
+    RNA-Seq:
+      class: File
+      location: https://zenodo.org/records/11367439/files/SRR8534859_RNASeq_mapped.bam
+      filetype: bam
+ 
+  
+  outputs:
+    genome_annotation_gtf:
+      location: https://zenodo.org/records/13941438/files/gffread.gtf?download=1
+      compare: sim_size
+      delta: 300000    
```

Comment:
As all the outputs are text files, please use asserts. For eg. check for number of lines, number of columns, check if specific lines exist (also with regex). Please do this for other outputs too.